blocks/header/header.css

```diff
@@ -34,7 +34,6 @@ header nav {
   display: flex;
   align-items: center;
   padding: 0 0 calc(var(--nav-padding) - 2px);
-  border-bottom: 2px solid var(--neutrals-20);
```

Comment:
@naina66406  Header needs border bottom, why its removed?

Reply:
<img width="624" alt="Screenshot 2024-09-02 at 5 02 06 PM" src="https://github.com/user-attachments/assets/0ea08e27-1811-4308-931e-ef76d54a79fc">
@mkbansal1 Noticing this on mobile. Maybe it needs some adjustments. Also, I couldn't find border spec in designs.

Reply:
Yes its not in figma